src/components/DpTreeList/DpTreeListNode.vue

```diff
@@ -41,11 +42,10 @@
         </div>
       </div>
       <dp-tree-list-toggle
-        data-cy="treeListChildToggle"
         v-if="isBranch"
-        class="self-start"
+        :class="alignToggle === 'top' ? 'self-start' : 'self-center'"
+        data-cy="treeListChildToggle"
         :disabled="!hasToggle"
-        v-tooltip="Translator.trans(!hasToggle ? 'no.elements.existing' : '')"
```

Comment:
Do we still need the import and directives in this file if we remove tooltip here?

Reply:
Thanks, they are not needed anymore! Removed in 2d3a6bcc776db038d3e010eb248d9083c3093ebe